oracle/config/shutdown-manager.config.js

```diff
@@ -0,0 +1,11 @@
+const baseConfig = require('./base.config')
+
+module.exports = {
+  ...baseConfig.bridgeConfig,
+  id: 'shutdown-manager',
+  name: 'shutdown-manager',
+  pollingInterval: 120000,
```

Comment:
Does it make sense to make this parameter configurable in order to avoid re-building of the docker image if it is necessary to change the interval?